lib/util/helpers.js

```diff
@@ -0,0 +1,21 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+var assert = require('assert');
+
+/**
+ * @param {Any} param The property to assert
+ * @param {String} message The failing message
+ * @returns {Error} an Error instance or null
+ */
+exports.doAssert = function(param, message) {
+  try {
+    assert(param, message);
+  } catch (err) {
+    return err;
+  }
+  return null;
+};
```

Comment:
If you intend to use this function rather than the suggestion above, then you'll need to edit it to call `doAssert` as well: https://github.com/strongloop/loopback-connector-couchdb2/pull/8/files#diff-98a56132bbbe76103bee832f4f76004aR319